From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig?= <uwe@kleine-koenig.org>
Date: Thu, 26 Oct 2023 13:49:36 +0200
Subject: [PATCH] Release 2023.03.0/customers/lxa/tac/20231026-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index af7af91112be..042f49d99065 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2023
 PATCHLEVEL = 03
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20231026-1
 NAME = None
 
 # *DOCUMENTATION*
